coderd/database/modelmethods.go

```diff
@@ -10,6 +10,8 @@ import (
 	"github.com/coder/coder/v2/coderd/rbac"
 )
 
+var DefaultDisplayApps = []DisplayApp{DisplayAppVscode, DisplayAppWebTerminal, DisplayAppSSHHelper, DisplayAppPortForwardingHelper}
```

Comment:
Is it used anywhere?

Reply:
Nah a relic from an earlier implementation